test/disability-benefits/components/DocumentRequestPage.unit.spec.jsx

```diff
@@ -11,5 +13,121 @@ describe('<DocumentRequestPage>', () => {
           loading/>
     );
     expect(tree.everySubTree('LoadingIndicator')).not.to.be.empty;
+    expect(tree.everySubTree('.claim-container')).to.be.empty;
+  });
+  it('should render upload error alert', () => {
+    const trackedItem = {
+      type: 'still_need_from_you_list',
+    };
+    const claim = {
+      id: 1
+    };
+    const tree = SkinDeep.shallowRender(
+      <DocumentRequestPage
+          trackedItem={trackedItem}
+          claim={claim}
+          uploadError/>
+    );
+    expect(tree.subTree('UploadError')).not.to.be.false;
+  });
+  it('should render due date info', () => {
+    const trackedItem = {
+      type: 'still_need_from_you_list',
+      suspenseDate: '2010-05-10'
+    };
+    const claim = {
+      id: 1
+    };
+    const tree = SkinDeep.shallowRender(
+      <DocumentRequestPage
+          claim={claim}
+          trackedItem={trackedItem}/>
+    );
+    expect(tree.subTree('DueDate')).not.to.be.false;
+    expect(tree.subTree('DueDate').props.date).to.eql(trackedItem.suspenseDate);
+  });
+  it('should render optional upload alert', () => {
+    const trackedItem = {
+      type: 'still_need_from_others_list',
+      suspenseDate: '2010-05-10'
+    };
+    const claim = {
+      id: 1
+    };
+    const tree = SkinDeep.shallowRender(
+      <DocumentRequestPage
+          claim={claim}
+          trackedItem={trackedItem}/>
+    );
+    expect(tree.subTree('.optional-upload')).not.to.be.false;
```

Comment:
I was trying to avoid checking content since it feels like that can/will change more often.